airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/publish/pipeline.py

```diff
@@ -431,9 +458,35 @@ async def _run(self) -> StepResult:
         return await super()._run()
 
 
+class CheckConnectorVersionIncrement(Step):
+    context: PublishConnectorContext
+    
+    @property
+    def title(self) -> str:
+        return "Check if connector version was incremented"
+
+    async def _run(self) -> StepResult:
+        # Skip if running from manual GitHub workflow
+        if os.getenv("GITHUB_WORKFLOW_MANUAL") == "true":
+            return StepResult(step=self, status=StepStatus.SKIPPED, stdout="Skipping version increment check for manual workflow run.")
```

Comment:
I'd say this should be SUCCESS with a comment that we override publish check and publish manually.